server/serverRun.py

```diff
@@ -174,50 +187,28 @@ def signup():
                 user = User(username=username, email=email, password_hash=password_hash)
                 db.session.add(user)
                 db.session.commit()
-                return json.dumps({
-                    'msg': 'User created'
-                }), 200, JSON_CT
+                return ok_response('User created')
             except Exception:
-                return json.dumps({
-                    'msg': 'Could not create user'
-                }), 400, JSON_CT
+                return INTERNAL_SERVER_ERROR_JSON_RESPONSE
 
-    return json.dumps({
-        'msg': 'Missing parameters'
-    }), 400, JSON_CT
+    return BAD_REQUEST_JSON_RESPONSE
 
 
 @app.route('/all_users', methods=['GET'])
 @logged_in(as_user='bugmommy')  # TODO create admin account
+@marshal_with(UserSchema(many=True))
 def all_users():
-    users = [
-        {
-            'id': user.id,
-            'username': user.username,
-            'email': user.email,
-            'password_hash': user.password_hash,
-            'create_date': str(user.create_date),
-            'num_items': len(Item.query.filter_by(user_id=user.id).all())
-        }
-        for user in User.query.all()
-    ]
-    return json.dumps(users, indent=2), 200, JSON_CT
+    return User.query.all()
 
 
 @app.route('/items', methods=['GET'])
 @logged_in()
+@marshal_with(ItemSchema(many=True))
 def items():
     username = session['username']
     user = User.query.filter_by(username=username).first()
     user_items = Item.query.filter_by(user_id=user.id).all() or []
-
-    user_items = [
-            {
-                'id': item.id
-            }
-            for item in user_items
-    ]
-    return json.dumps(user_items), 200, JSON_CT
+    return user_items
 
 
 @app.route('/item/<int:item_id>',methods=['DELETE'])
```

Comment:
Forgot to change this

Reply:
This is a rather large change to master without any real evidence of a working build. Can you please attach screenshots of your change. Otherwise I look forward to using schemas in future sprints